files/ansible/tasks/call_harness.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Set CAP_SYS_ADMIN to 2
+  shell: echo '2' | tee /proc/sys/kernel/perf_event_paranoid
```

Comment:
I'm thinking, should we save the old state and restore after calling the harness?
At least checking if the flag is already 2 would avoid the need for the user to be root every time.